semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/structure/MethodParameter.scala

```diff
@@ -10,39 +10,39 @@ import io.shiftleft.semanticcpg.language.types.propertyaccessors._
 /**
   * Formal method input parameter
   * */
-class MethodParameter(raw: GremlinScala[nodes.MethodParameterIn])
-    extends NodeSteps[nodes.MethodParameterIn](raw)
-    with EvalTypeAccessors[nodes.MethodParameterIn] {
+class MethodParameter(val wrapped: NodeSteps[nodes.MethodParameterIn])
+    extends EvalTypeAccessors[nodes.MethodParameterIn] {
```

Comment:
Why do we need to extend from EvalTypeAccessors here? There should be a separate pimp class with its on implicit conversion.

Reply:
agreed, and it's on my list already, but that'll make it into a follow-up PR.